app/assets/stylesheets/front/pages/meditations.sass

```diff
@@ -1,132 +1,77 @@
 @import "../../mixins/vars"
 
 body.meditations-index
-
   &.open-menu
     @include tablet-or-smaller
       overflow: hidden
       height: 100vh
 
-  main section
-    padding: 20px 0
-    margin-bottom: 0
-    
-    &.page-title
-      padding: 25px 0
-      
+  section.intro
+    margin: 0
+
     .title
-      text-align: center
-      font-size: 2.125em
+      margin-top: 1em
+      font-size: 2.5em
       font-family: $raleway-bold
-      margin: 0
-      letter-spacing: .05em
-
-      @include desktop
-        font-size: 2.8125em
-        margin: 24px 0
+      line-height: 150px
 
-      @include tablet-or-smaller
-        font-size: 1.675em
+      display: flex
+      align-items: center
+      justify-content: center
 
-      @include mobile
-        line-height: 1.25em
-        
-    h2.title
-      font-family: $raleway-medium
-      font-weight: 500
-      letter-spacing: .02em
-      margin: 35px 0 25px
+      &:after
+        width: 100%
+        left: auto
+        right: 50%
```

Comment:
Here we are changing to make the header use our generic `section.intro` tag. Which means less code is needed here to special case this page.